app/Models/GeneralAssemblies/GeneralAssembly.php

```diff
@@ -218,4 +220,23 @@ public static function isTemporaryPasscode(string $value): bool
         return $value == self::getTemporaryPasscode()
             || $value == self::getTemporaryPasscode('-1 minute');
     }
+
+    /**
+     * The "booting" method of the model.
+     * Initializes the excused users with the current semester's passive users.
+     *
+     * @return void
+     */
+    protected static function boot(): void
+    {
+        parent::boot();
+
+        static::created(function ($general_assembly) {
+            // Excuse passive students. They are excused at the creation of the general assembly (as opposed to when it
+            // is opened) because the list of excused students is displayed as soon as the general assembly is created.
+            foreach (Semester::current()->usersWithStatus(SemesterStatus::PASSIVE)->get() as $user) {
+                $general_assembly->excusedUsers()->attach($user);
```

Comment:
I believe the attach can take a collection of users in once, so you don't need the foreach.